substrate/utils/frame/benchmarking-cli/src/pallet/writer.rs

```diff
@@ -467,7 +467,10 @@ pub(crate) fn write_results(
 				file_name = format!("{}_{}", file_name, instance.to_snake_case());
 			}
 			// "mod::pallet_name.rs" becomes "mod_pallet_name.rs".
-			file_path.push(file_name.replace("::", "_"));
+			file_name = file_name.replace("::", "_");
+			// Some old runtimes have a bug with the pallet and instance name containing a space
```

Comment:
@karolk91 a complement to your: https://github.com/paritytech/polkadot-sdk/pull/9354?